lib/matplotlib/axes/_axes.py

```diff
@@ -2691,78 +2691,84 @@ def errorbar(self, x, y, yerr=None, xerr=None,
                  barsabove=False, lolims=False, uplims=False,
                  xlolims=False, xuplims=False, errorevery=1, capthick=None,
                  **kwargs):
-        """
-        Plot an errorbar graph.
-
-        Call signature::
-
-          errorbar(x, y, yerr=None, xerr=None,
-                   fmt='', ecolor=None, elinewidth=None, capsize=None,
-                   barsabove=False, lolims=False, uplims=False,
-                   xlolims=False, xuplims=False, errorevery=1,
-                   capthick=None)
-
-        Plot *x* versus *y* with error deltas in *yerr* and *xerr*.
-        Vertical errorbars are plotted if *yerr* is not *None*.
-        Horizontal errorbars are plotted if *xerr* is not *None*.
+        """Plot an errorbar graph.
```

Comment:
This should remain on a separate line; I think this is causing the rendering issue @NelleV noted.